ci/kokoro/docker/build-in-docker-cmake.sh

```diff
@@ -129,6 +129,7 @@ if [[ "${CLANG_TIDY:-}" == "yes" && (\
   # w.r.t. the target branch.
   echo
   log_yellow "Running clang-tidy on presubmit build, only changed files are tested."
+  ${CMAKE_COMMAND} --build "${BINARY_DIR}" --target nlohmann_json_project
```

Comment:
Q: Rather than running a separate cmake build here, could this block be moved down about 10 lines until after we finish the normal build? It looks like maybe that'd work? Just wondering.

Reply:
I wanted the `clang-tidy` results early, I find it frustrating to wait for a full build and then discover that the formatting / clang-tidy or something like that is broken.

But I will gladly approve a PR if you or anybody else in the team feels strongly enough to make the changes.

Reply:
That reason makes sense to me. Thanks.